packages/core/src/components/Multicomplete.story.tsx

```diff
@@ -48,4 +48,26 @@ storiesOf('Core/Multicomplete', module)
       renderItem={(item, highlighted, selected) => <Text bold={selected}>{item.name}</Text>}
       value={['red', 'green']}
     />
+  ))
+  .add('Load items on focus.', () => (
+    <Multicomplete
+      accessibilityLabel="Favorite color?"
+      label="Favorite color?"
+      name="autocomplete"
+      onChange={action('onChange')}
+      onSelectItem={action('onSelectItem')}
+      onLoadOptions={value =>
+        Promise.resolve(
+          [
+            { value: 'red', name: 'Red' },
+            { value: 'black', name: 'Black' },
+            { value: 'blue', name: 'Blue' },
+            { value: 'green', name: 'Green' },
+          ].filter(item => !value || item.name.toLowerCase().match(value.toLowerCase())),
+        )
+      }
+      renderItem={(item, highlighted, selected) => <Text bold={selected}>{item.name}</Text>}
+      value={['red', 'green']}
+      loadItemsOnFocus
```

Comment:
can you add a screenshot of this working for Multicomplete too?

Reply:
done (above)